pkg/sca/sca_test.go

```diff
@@ -152,12 +160,18 @@ func TestExecableSharedObjects(t *testing.T) {
 
 	want := config.Dependencies{
 		Runtime: []string{
+			"so-ver:ld-linux-aarch64.so.1",
+			"so-ver:libc.so.6",
+			"so-ver:libcap.so.2",
+			"so-ver:libpsx.so.2",
```

Comment:
to fix abi coupling, wasn't we supposed to generate versioned depends here? As in depends on "so-ver:libc.so.6~2.41" ? (or >=2.41, or >=2.41-r0, or somesuch)

Reply:
Yeah, this is still a WIP and I fixed it locally.  Let me force-push what I have.